packages/node_modules/@webex/plugin-meetings/src/meeting/index.js

```diff
@@ -1273,6 +1273,12 @@ export default class Meeting extends StatelessWebexPlugin {
   setUpLocusInfoMeetingListener() {
     this.locusInfo.on(EVENTS.REMOTE_ANSWERED_DECLINED, (payload) => {
       this.meetingFiniteStateMachine.remote(payload);
+
+      this.leave({reason: payload.reason}).then(() => {
+        LoggerProxy.logger.warn('Meeting:index#setUpLocusInfoMeetingListener --> REMOTE_ANSWERED_DECLINED. The meeting has been left and has been destroyed.');
```

Comment:
@arun3528 should this be a warning? It is the expected case. I would assume just an `info`.

Reply:
I copied this over from the if block below it. It may not align since that doesn't actually destroy the meeting.
So i think an `info` works out

Reply:
It can be an `info` but since we're saying `Attempting` it can still be a `warn`

Reply:
its expected so it think its an info but , as user might not know why the meeting ended on there side if they are not listning to declined event then we need this 

@taymoork2  do the developer get a declined event on there side ?